code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -20,14 +22,21 @@
 	armor = 20 // 120 damage to KO a zombie, which kills it
 	speedmod = 2
 	mutanteyes = /obj/item/organ/eyes/night_vision/zombie
+	var/regen_cooldown = 0
 
 /datum/species/zombie/infectious/spec_stun(mob/living/carbon/human/H,amount)
 	. = min(2, amount)
 
+/datum/species/zombie/infectious/apply_damage(damage, damagetype = BRUTE, def_zone = null, blocked, mob/living/carbon/human/H)
+	. = ..()
+	if(.)

```

Comment:
this seems unnecessary

Reply:
Incorrect, wrong, please do not do this to any pr ever agai

Reply:
Please don't bully him...

Reply:
m-m-m-me or robustin?

Reply:
What do you mean? Ah, so you were bullying Robustin?
Very bad behavior. RIP, you.

Reply:
I was referring to Joan's statement about being wrong.

Reply:
I believe @ChangelingRain was the one who told me to set it up like this, but I'm a little foggy... if she's the one who suggested it then it would made sense that @ExcessiveUseOfCobblestone was the one being cyberbullied.

Reply:
i mean after looking at it I'm assuming what it's doing is seeing if inher returns TRUE and if it does only then apply the cooldown, although they could have just said that instead of being salty.

If that's the case though why can't you just do `if(..())` instead of `. = ..() /n if(.)`